mm2src/mm2_bitcoin/chain/src/transaction.rs

```diff
@@ -229,6 +229,7 @@ pub struct Transaction {
     /// https://github.com/navcoin/navcoin-core/blob/556250920fef9dc3eddd28996329ba316de5f909/src/primitives/transaction.h#L497
     pub str_d_zeel: Option<String>,
     pub tx_hash_algo: TxHashAlgo,
+    pub v_extra_payload: Option<Vec<u8>>,
```

Comment:
Please add a comment above this that links to https://github.com/firoorg/firo/blob/bf7c7fa555bd00db970c5c3e64e7452ef20c800e/src/primitives/transaction.h#L389